happiness-backend/api/dao/happiness_dao.py

```diff
@@ -68,3 +68,6 @@ def get_happiness_by_group_count(user_ids, page, n):
 def get_paginated_happiness_by_query(user_id, query, page, n):
     return Happiness.query.filter_by(user_id=user_id) \
         .filter(Happiness.comment.like(f"%{query}%")).paginate(page=page, per_page=n, error_out=False)
+
+def get_comment_by_id(id):
+    return Comment.query.filter_by(id=id).first()
```

Comment:
Do not add legacy SQLAlchemy code to the codebase. Please use version 2.0 SQLAlchemy queries.